backlogger.py

```diff
@@ -5,7 +5,8 @@
 from datetime import datetime, timedelta
 from inspect import getmembers, isfunction
 import requests
-from requests.packages.urllib3.util.retry import Retry
+from urllib3.util.retry import Retry
+from urllib.parse import urlparse
```

Comment:
Hrm. Why didn't CI on #11 fail, though?

Reply:
You can **see** the preview

Reply:
I wonder if that could be due to #13 and it never used the latest branch

Reply:
The preview for #11 was generated inside https://github.com/openSUSE/backlogger/actions/runs/3704360249 which is using the code from the main branch?